### ISSUE TITLE
Publishing a book with DocPad

### ISSUE THREAD

**Issue by ghost** [docpad/docpad#832]
I'm writing a documentation book that I want to publish through Leanpub, Amazon, etc.
Because the book will be published under [GNU Free Documentation License](http://www.gnu.org/copyleft/fdl.html) I also want to publish in a website.

I'm writing the book in markdown, I was taking a look at docpad and I think it's a great  tool to help me to publish the book in a website because it's easy to use, easy to customize and I can just copy the markdown files to the document folder, add a customized layout and it's automatically rendered. That's great.

My problem is that I want to use the same markdown files to publish the digital book and the "web-book" without the need of changing anything from them.

In short, that means that I want that Docpad automatically do this for me:

Source: `src/documents/example.md`

``` markdown
# Lorem

pure markdown file
```

On the fly, Docpad convert source to a template file (e.g. eco) and preppend the header with variables:

``` markdown

---
layout: "default"
isPage: true
----

# Lorem

pure markdown file
```

Finally, render that to a html file.

Is it possible that Docpad can do that for me? Any guide to do it?
